README.md

```diff
@@ -3,7 +3,7 @@ sbt-mocha
 
 [![Build Status](https://api.travis-ci.org/sbt/sbt-mocha.png?branch=master)](https://travis-ci.org/sbt/sbt-mocha)
 
-Allows mocha to be used from within sbt.
+Allows mocha to be used from within sbt via the use of [sbt-web](https://github.com/sbt/sbt-web). You can therefore run your JS tests using SBT conventions either in-JVM (eg Trireme), or on (Node).
```

Comment:
I don't think you can run on Trireme anymore

Reply:
npm won't run on Trireme, but mocha I think should.

Reply:
We're getting lots of bug reports right now of sbt-less being broken if you don't have node installed. I don't have time to investigate, but it kind of looked like sbt-web in general was broken with Trireme